### ISSUE TITLE
Collaboration for hackathon

### ISSUE THREAD

**Issue by BeepLoveKarki** [openjs-foundation/cross-project-council#572]
Hello, I just wanted to have a collaborative talk or plan with the committee for collaboration for an international digital hackathon named QuantunHack (https://quantumhack.tech) licensed under global brand of Major League Hacking (https://mlh.io). The date for the event is July 31-Aug 1.

Thanks, waiting for response asap

**Comment by joesepi** [comment]
Hi @BeepLoveKarki. Thanks for reaching out. Can you be more clear about what it is you'd like from the Cross Project Council, the Foundation or any of its members? Thanks!

**Comment by BeepLoveKarki** [comment]
Thank you so much @joesepi for the quick reply. I wanted any form of partnership or support in any form to collaborate together to make the event grand since openjs-foundation is one of the largest tech community in the world.

I here attach you with the event prospectus and partnership benefits docs. You may have a look and say on under which criteria you may help us and we may help you.

Event & Sponsorship Prospectus:  https://drive.google.com/file/d/1ppDcEw-aNCyWpsZSZTSCadMdUyJUPOU_/view?usp=sharing

Mutual Partnership Benefits: https://docs.google.com/document/d/1Pkr0lBOG44DYV6h903HEZNakLxDqW7fZoiKOSFLv96A/edit?usp=sharing

**Comment by joesepi** [comment]
Hello @BeepLoveKarki. I think sponsorship is unlikely, but is a topic we could discuss in our next meeting.

However, if you had other concrete, tangible ways that you would like to for people and organizations to engage with your event, please share those and we can discuss those as well. I believe in Slack, you mentioned mentorship and judging. If you could elaborate in some detail, perhaps people in our community could consider getting involved.

Thanks!

**Comment by BeepLoveKarki** [comment]
Yes sure. I just shared you the docs. If this community can help me find best mentors, judges and even participants, we are ok to work like a community partner for the event. 😊

**Comment by BeepLoveKarki** [comment]
Mentors can be one with good tech skills and judges the ones with good experience and skills 😊

**Comment by BeepLoveKarki** [comment]
Hey @joesepi sorry for late reply, I am here with the exact two way support key points between openjs and our event committee.

https://docs.google.com/document/d/17KxRJP7R4ZFZRZUuqvBgI-qzGLgA_gjpmQuWTc-WrDc/edit?usp=sharing

This is just a simple outline, we may plan more or go for official MoU or agreement after both sides satisfaction. 😊

**Comment by joesepi** [comment]
For others to easily consume the details:


# Technical Partner Collaboration

## On Behalf of OpenJS Foundation

- Provide 2 Mentors for the hackathon
- Provide 2 Judges for the hackathon
- Provide APIs, credits or any programming tools possible
- Promote hackathon via their network and media platforms
- Sponsor NodeJs Certification to winner(s) or qualified enthusiast team member(s)

## On Behalf of QuantumHack

- Announcement as one of the Partners on social media
- Announcement as one of the partner during the event opening and closing of the hackathon
- Logo in our flyers
- Logo on the website
- Any workshop, training or competition conduction from OpenJS team if wished
- Access to participants information(i.e mail) if wished


*Note: The terms can be negotiated*

**Comment by BeepLoveKarki** [comment]
Yes. Thanks. Hope to have some response as soon as possible. 😊

**Comment by BeepLoveKarki** [comment]
Hey @joesepi , any response or update?

**Comment by joesepi** [comment]
Date is now AUGUST 28 - 29

**Comment by designMoreWeb** [comment]
I choose to volunteer for mentoring
